bve-corpus/src/bin/bve-parser-run.rs

```diff
@@ -138,10 +140,34 @@ fn parse_config_extensions_cfg(file: impl AsRef<Path>, options: &Options) {
     }
 }
 
+fn parse_config_ats_cfg(file: impl AsRef<Path>, options: &Options) {
+    let contents = read_convert_utf8(file).expect("Must be able to read file");
+
+    let start = Instant::now();
+    let (parsed, warnings) = parse_ats_cfg(&contents);
+    let duration = Instant::now() - start;
+
+    println!("Duration: {:.4}", duration.as_secs_f32());
```

Comment:
This is fine for now, but you'll probably eventually want to use more of a profiling framework

Reply:
Yeah, I was just thinking about how bad that code is getting and it desperately needs a refactor.